packages/shared/styles/components/SfHero.scss

```diff
@@ -0,0 +1,48 @@
+@import '../variables';
+@import '~@glidejs/glide/src/assets/sass/glide.core';
+
+$carousel-mobile-breakpoint: $mobile-max;
```

Comment:
again, the SCSS variables should be prefixed by respective component name, so it should be `$hero-mobile-breakpoint`